packages/elements-core/src/components/TryIt/TryIt.spec.tsx

```diff
@@ -811,12 +828,12 @@ describe('TryIt', () => {
 
     describe('OAuth2 Component', () => {
       it('allows to send a OAuth2 request', async () => {
-        render(<TryItWithPersistence httpOperation={putOperation} />);
+        const { getByLabelText } = render(<TryItWithPersistence httpOperation={putOperation} />);
```

Comment:
Why do you keep switching up `screen.doSomething` for this?

Reply:
Good question - as I was fixing the tests I briefly thought that perhaps the screen thing was causing an issue, so I switched it and did not switch it back. Might not be the case, I'll switch it back NP.